docs/main/concepts/architecture.md

```diff
@@ -166,4 +166,4 @@ Check out our [troubleshooting section](../troubleshooting/) for some of the com
 
 - [Data Engines](../concepts/data-engines/data-engines.md)
 - [OpenEBS Local Storage](../concepts/data-engines/local-storage.md)
-- [OpenEBS Replicated Storage](../concepts/data-engines/replicated-engine.md)
+- [OpenEBS Replicated Storage](../concepts/data-engines/replicated-storage.md)
```

Comment:
Wasn't this made in an earlier PR? You need to rebase

Reply:
This change was not there, Vishnu. I checked.